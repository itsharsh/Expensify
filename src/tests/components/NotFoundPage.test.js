import React from "react";
import { shallow } from "enzyme";

import NonFoundPage from "../../components/NotFoundPage";

test("should render NotFoundPage Component correctly", () => {
  const wrapper = shallow(<NonFoundPage />);
  expect(wrapper).toMatchSnapshot();
});
