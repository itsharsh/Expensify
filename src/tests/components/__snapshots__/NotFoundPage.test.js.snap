// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPage Component correctly 1`] = `
<div>
  <h1>
    404 Page not found
  </h1>
  <Link
    to="/"
  >
    Go Home
  </Link>
</div>
`;
